AppCenter/AppCenterTests/MSHttpUtilTests.m

```diff
@@ -128,6 +128,21 @@ - (void)testSSLConnectionErrorDetected {
   XCTAssertFalse([MSHttpUtil isSSLConnectionError:error]);
 }
 
+- (void)testHideSecretInString {
+    
+    // If
+    NSString *secret = @"12345678-1234-1234-1234-123456789012";
+    NSString *string = [NSString stringWithFormat:@"this-%@-should-be-encode", secret];
+    NSString *expectedEncodeString = @"this-****************************56789012-should-be-encode";
```

Comment:
```suggestion
    NSString *expectedEncodeString = @"this-****************************56789012-should-be-encoded";
```

Reply:
Maybe there is a way to construct string from amount of '*' equal to `secret.length` ?